contracts/IPresale.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.9;
+
+interface IPresale {
+    event ConfigUpdated(PresaleConfig prevConfig, PresaleConfig newConfig, address indexed sender);
+
+    event RoundsUpdated(RoundConfig[] prevRounds, RoundConfig[] newRounds, address indexed sender);
+
+    event Receipt(
+        uint256 indexed roundIndex,
+        uint256 tokenPrice,
+        address indexed asset,
```

Comment:
lets move asset to fist item in the event. (generally indexed items are first). But I usually keep sender at the end.